.storybook/.browserslistrc

```diff
@@ -1,4 +1,4 @@
 last 2 Chrome versions
 last 2 FF versions
 last 2 Edge versions
-Safari >= 12
+Safari >= 15
```

Comment:
Did this manage to actually fix anything?

Reply:
No :D but I think it's still okay to do that.